### PR TITLE
[REVIEW] upgrade to RAPIDS 0.9"

### PR DESCRIPTION
**Pull request by yidong72**
Fixed a bunch of issues in reverted review #57

### CODE REVIEW

**Comment by avolkov1** [comment]
Can you resolve conflicts on this merge?
https://github.com/rapidsai/gQuant/pull/60/conflicts

I'm reviewing it right now. Looks good. Just testing.

File: docker/build.sh [hunk 1]
```diff
@@ -70,7 +74,11 @@ SHELL ["bash","-c"]
 # Additional python libs
 #
 RUN source activate rapids \ 
-    && pip install nxpd $CUPY
+    && pip install nxpd $CUPY 
+
+RUN source activate rapids \
+    && cd /rapids/gQuant \
```
[the remark this answers is not in the document]
Comment (yidong72) [reply docker/build.sh]: yeah. I fixed it locally. I forget to push it

**Comment by yidong72** [comment]
It is weird.

I checked  

https://github.com/rapidsai/gQuant/pull/60/conflicts

line 41
'''
CONTAINER="nvcr.io/nvidia/rapidsai/rapidsai:cuda${CONTAINER_VER}-runtime-ubuntu${OS_STR}"
'''

But my branch at line 41 is 
https://github.com/yidong72/gQuant/blob/226256bb937fa0d9463cd208e63de4c80968e3fa/docker/build.sh#L41

If we merge it, the line41 will be lost.

**Comment by yidong72** [comment]
Let's close this PR and use [PR61](https://github.com/rapidsai/gQuant/pull/61) instead. It has the same contents as this branch head but based on the original develop branch.

**Pull request closed without merging**